src/Nethermind/Nethermind.Evm.Test/Eip3588Tests.cs

```diff
@@ -35,7 +35,8 @@ namespace Nethermind.Evm.Test
     [TestFixture]
     public class Eip3855Tests : VirtualMachineTestsBase
     {
-        protected override long BlockNumber => MainnetSpecProvider.ShanghaiBlockNumber;
+        protected override long BlockNumber => MainnetSpecProvider.GrayGlacierBlockNumber;
```

Comment:
Should we add `MainnetSpecProvider.ShanghaiBlockNumber` alias with same block number?
Should we add static `ForkActivation`s to `MainnetSpecProvider` ?

Reply:
Many changes???

Reply:
So my idea was instead of keeping them as `const long` change them to `static ForkActivation`. Not sure if worth it?

Reply:
yup, I agree with Łukasz but it depends on how many changes we need to do